packages/gatsby/cache-dir/static-entry.js

```diff
@@ -207,7 +207,7 @@ export default (pagePath, callback) => {
       key={`webpack-runtime`}
       id={`webpack-runtime`}
       dangerouslySetInnerHTML={{
-        __html: runtimeRaw,
+        __html: `/*<![CDATA[*/${runtimeRaw} /*]]>*/`,
```

Comment:
We did remove inlining of webpack runtime here - https://github.com/gatsbyjs/gatsby/pull/5540